src/renderer/components/dex/OrderForm.vue

```diff
@@ -269,21 +269,30 @@ export default {
       }
       return !this.$store.state.orderQuantity || !this.$store.state.orderPrice || this.$isPending('placeOrder');
     },
+    orderTypes() {
+      const list = [{
+        label: 'Limit',
+        value: 'Limit',
+      },
+      ];
+      if (this.canPlaceMarketOrder === true) {
```

Comment:
You don’t need these explicit true/false comparisons throughout unless the variable could be someone other than Boolean/null/undefined.